docker/php/docker-entrypoint

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+# first arg is `-f` or `--some-option`
+if [ "${1#-}" != "$1" ]; then
+	set -- php-fpm "$@"
+fi
+
+if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
+  composer install
```

Comment:
This runs `composer install` when the container(s) start?

Reply:
Right. This runs inside the php container on startup. As the application directory is mounted as a volume, `vendor/`-directory is persisted.